static/js/click_handlers.js

```diff
@@ -138,6 +138,14 @@ $(function () {
         meta.focusing = true;
     });
 
+    // MUTING
+
+    $('body').on('click','.on_hover_topic_mute', function (e) {
+        var stream = $(e.currentTarget).attr('data-stream-name');
+        var topic = $(e.currentTarget).attr('data-topic-name');
```

Comment:
this should probably call `e.stopPropagation` like the other similar click handlers.